PS8lite.Rmd

```diff
@@ -201,9 +226,17 @@ Fit one Random Forest model `model_rf` using any approach you want.
 Using `model_rf` make predictions and submit to Kaggle.
 
 ```{r}
-
+test <- test %>% 
+  mutate(
+    logSalePrice_hat_rf_caret = predict(model_rf_caret, test),
+    SalePrice_hat_rf_caret = exp(logSalePrice_hat_rf_caret) -1
+  )
+submission_rf_caret <- test %>%
+  mutate(SalePrice = SalePrice_hat_rf_caret) %>%
+  select(Id, SalePrice)
+write_csv(submission_rf_caret, "data/submission_rf_caret.csv")
 ```
 
 
-![](images/score_screenshot_naive.png){ width=100% }
+![](images/score_screenshot_random_forest.png){ width=100% }
```

Comment:
Hi, everything looks good. Here is an example comment that I would leave as part of a “code review” if this were real life. Then you would make the requested changes to this PR and then I would merge your changes into my master copy of the repo. I won't be doing that last step in this case.